tomland.cabal

```diff
@@ -122,7 +123,7 @@ benchmark tomland-bench
   hs-source-dirs:     benchmark
   main-is:            Main.hs
 
-  other-modules:      Benchmark.Compare
+  other-modules:      Benchmark.Main
```

Comment:
It shouldn't be `Benchmark.Main`. Couple lines above you have: `main-is: Main.hs` so you can put everything from `Benchmark.Main` module to `Main.hs` module.